### ISSUE TITLE
Running makemigrations is trying to generate a migration for socialaccount app

### ISSUE THREAD

**Issue by YuriHeupa** [pennersr/django-allauth#1336]
When i run makemigrations for my project it's trying to generate a new migration in sociallaccount app, with the following content:

```
# -*- coding: utf-8 -*-
from __future__ import unicode_literals

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('socialaccount', '0003_extra_data_default_dict'),
    ]

    operations = [
        migrations.AlterField(
            model_name='socialaccount',
            name='provider',
            field=models.CharField(max_length=30, verbose_name='provider', choices=[(b'facebook', b'Facebook')]),
        ),
        migrations.AlterField(
            model_name='socialapp',
            name='provider',
            field=models.CharField(max_length=30, verbose_name='provider', choices=[(b'facebook', b'Facebook')]),
        ),
    ]
```

Is normal or it's a bug?

**Comment by s4291158** [comment]
all they needed was orderdict

**Comment by pennersr** [comment]
Closed by #1534

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/1336
